week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,12 +12,27 @@
  */
 
 class ATM {
-    // Add your code here
-
+    constructor () {
+        this.accountBalance = 100;
+        this.deposit = 0;
+        return this.accountBalance;
```

Comment:
the constructor is a function but you don't need to return anything as it can't be called from outside